chef/data_bags/crowbar/migrate/nova/032_add_zvm_attributes.rb

```diff
@@ -0,0 +1,16 @@
+def upgrade ta, td, a, d
+  a['zvm'] = ta['zvm']
+  d['element_states'] = td['element_states']
+  d['element_order'] = td['element_order']
+  d['element_run_list_order'] = td['element_run_list_order']
+  return a, d
+end
+
+def downgrade ta, td, a, d
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `ta`. If it's necessary, use `_` or `_ta` as an argument name to indicate that it won't be used.<br>Lint/UnusedMethodArgument: Unused method argument - `td`. If it's necessary, use `_` or `_td` as an argument name to indicate that it won't be used.<br>Style/MethodDefParentheses: Use def with parentheses when there are parameters.